org.KekikAkademi.eFatura.yml

```diff
@@ -0,0 +1,52 @@
+# Bu araç @keyiflerolsun tarafından | @KekikAkademi için yazılmıştır.
+
+app-id: org.KekikAkademi.eFatura
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: eFaturaGUI
+
+finish-args:
+  # ? X11 + XShm Erişimi
+  - --share=ipc
+  # ? Wayland Erişimi
+  - --socket=wayland
+  # ? Wayland mevcut olmadığında X11 Erişimi Ver
+  - --socket=fallback-x11
+  # ? Ağ Erişimi
+  - --share=network
+  # ? PulseAudio Kullanarak Sesleri Çal
+  - --socket=pulseaudio
+  # ! tesseract Erişim
+  - --env=TESSDATA_PREFIX=/app/share/tessdata
+
+# ! tesseract Çöpü Temizle
+cleanup:
+  - /sbin
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /bin/protoc
+  - /share/doc
+  - /share/pkgconfig
+  - /share/man
+  - /share/cmake*
+  - "*.a"
+  - "*.la"
+
+modules:
+- SRC/tesseract.yaml
```

Comment:
this is the same comment as previous reviews. I still applies. There is no reason make maintenance harder.

Reply:
https://github.com/flathub/flathub/pull/4169/commits/7da4cb0086287f88ab3089ed4dbf135694370a7b